jobs/openshift4/cluster/integreatly/openshift-4-cluster-integreatly-install.yaml

```diff
@@ -0,0 +1,61 @@
+---
+- job:
+    name: openshift-4-cluster-integreatly-install
+    display-name: 'Openshift 4 Cluster Integreatly Install'
+    project-type: pipeline
+    concurrent: true
+    parameters:
+        - string:
+            name: CLUSTER_DOMAIN
+            default: 'skaletaa.intlyqe.com'
+            description: "e.g. [cluster-name].open.redhat.com"
+        - string:
+            name: ADMIN_USERNAME
+            default: 'kubeadmin'
+            description: "Admin username for the OCP4 cluster"
+        - string:
+            name: ADMIN_PASSWORD
+            default: ''
+            description: "Admin password for OS4 cluster"
+        - bool:
+            name: CLEANUP
+            default: false
+            description: "Indicates whether to cleanup before execution"
```

Comment:
OK, since you added this we can leave the cleanup stage for now, it should be handled by a separate pipeline eventually though